chromium_src/BUILD.gn

```diff
@@ -414,4 +413,8 @@ source_set("chrome_spellchecker") {
     "//components/spellcheck/common",
     "//components/spellcheck/renderer",
   ]
+
+  if (enable_builtin_spellchecker) {
+    public_deps += [ "//chrome/common" ]
```

Comment:
I might be reading the deps, it looks like pref_names.h comes in from `//chrome/common:constants`. Would it make sense to use that instead of `//chrome/common` here to pick up the various `#includes` deps?

Reply:
☝️ In agreement, we shouldn't depend on all of `//chrome/common` here

Reply:
Yes, you are right, my bad. Following up with a fixup.